terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,6 @@
+environment = "cidev"
+aws_profile = "development-eu-west-2"
+
+# service configs
+log_level = "debug"
```

Comment:
We can remove the references to log_level in all the profiles - ideally we just want to define the log level in the ecs-configs env file instead

Reply:
done